jans-cedarling/bindings/cedarling_python/src/config/bootstrap_config.rs

```diff
@@ -5,520 +5,160 @@
-    /// the scheme must be https.
-    pub idt_iss_validation: String,
-
-    /// When enabled, the `sub` (Subject) claim must be present in the ID Token.
-    pub idt_sub_validation: String,
-
-    /// When enabled, the `exp` (Expiration) claim must be present in the ID Token
-    /// and not past the date specified.
-    pub idt_exp_validation: String,
-
-    /// When enabled, the `iat` (Issued at) claim must be present in the ID Token.
-    pub idt_iat_validation: String,
-
-    /// When enabled, the `aud` ( Audience) claim must be present in the ID Token.
-    pub idt_aud_validation: String,
-
-    /// When enabled, the `iss` (Issuer) claim must be present in the Userinfo Token and
-    /// the scheme must be https.
-    pub userinfo_iss_validation: String,
-
-    /// When enabled, the `sub` (Subject) claim must be present in the Userinfo Token.
-    pub userinfo_sub_validation: String,
-
-    /// When enabled, the `aud` (Audience) claim must be present in the Userinfo Token.
-    pub userinfo_aud_validation: String,
-
-    /// When enabled, the `exp` (Expiration) claim must be present in the Userinfo Token
-    /// and not past the date specified.
-    pub userinfo_exp_validation: String,
-
-    /// Varying levels of validations based on the preference of the developer.
-    ///
-    /// Could be set to either `'none'` or `'strict'`.
-    ///
-    /// # Strict Mode
-    ///
-    /// Strict mode requires:
-    ///     1. id_token aud matches the access_token client_id;
-    ///     2. if a Userinfo token is present, the sub matches the id_token, and that
-    ///         the aud matches the access token client_id.
-    pub id_token_trust_mode: String,
-
-    /// If Enabled, the Cedarling will connect to the Lock Master for policies,
-    /// and subscribe for SSE events.
-    pub lock: String,
-
-    /// URI where Cedarling can get JSON file with all required metadata about
-    /// Lock Master, i.e. .well-known/lock-master-configuration.
-    ///
-    /// ***Required*** if `LOCK == Enabled`.
-    pub lock_master_configuration_uri: Option<String>,
-
-    /// Controls whether Cedarling should listen for SSE config updates.
-    pub dynamic_configuration: String,
-
-    /// SSA for DCR in a Lock Master deployment. The Cedarling will validate this
-    /// SSA JWT prior to DCR.
-    pub lock_ssa_jwt: Option<String>,
-
-    /// How often to send log messages to Lock Master (0 to turn off trasmission).
-    pub audit_log_interval: u64,
-
-    /// How often to send health messages to Lock Master (0 to turn off transmission).
-    pub audit_health_interval: u64,
-
-    /// How often to send telemetry messages to Lock Master (0 to turn off transmission).
-    pub audit_health_telemetry_interval: u64,
-
-    // Controls whether Cedarling should listen for updates from the Lock Server.
-    pub listen_sse: String,
+    inner: cedarling::BootstrapConfig,
+    source: Option<cedarling::BootstrapConfigRaw>,
```

Comment:
Why we need both `cedarling::BootstrapConfig` and `cedarling::BootstrapConfigRaw`. 
I think we can have only `cedarling::BootstrapConfigRaw`

Reply:
when we load from a file using `cedarling::BootstrapConfig::load_from_file` it returns a `cedarling::BootstrapConfig` and we don't really have access to the struct that was used to deserialize it anymore.

Reply:
removed `BootstrapConfigRaw` in [3c383ea](https://github.com/JanssenProject/jans/pull/10350/commits/3c383eacce4deba2e1dd57e5e9f87c5496a2f67d).